payments-core/src/test/java/com/stripe/android/cards/CardAccountRangeServiceTest.kt

```diff
@@ -103,23 +103,24 @@ class CardAccountRangeServiceTest {
     }
 
     @Test
-    fun `If CBC is disabled, return the matched brand as soon as there is input`() = runTest {
-        val expectedAccountRange = AccountRange(
-            binRange = BinRange(
-                low = "4000000000000000",
-                high = "4999999999999999",
-            ),
-            panLength = 16,
-            brandInfo = AccountRange.BrandInfo.Visa,
-        )
+    fun `If CBC is disabled and only one brand matches, return the matched brand as soon as there is input`() =
```

Comment:
this test depends on there being only one card that matches the input. Now that the Cartes Bancaires test card also matches a "4" prefix, I needed to change the digit and card brand being used for this test